src/glue/jobs/s3_to_json.py

```diff
@@ -554,10 +555,10 @@ def write_file_to_json_dataset(
                 delete_upon_successful_upload=delete_upon_successful_upload
         )
         uploaded_files.append(current_output_path)
-        logger.info(
-                "Output file attributes",
-                extra={
-                    **logger_context,
+        logger_extra = dict(
+            merge_dicts(
```

Comment:
Is this something we should make sure to do for everywhere that we are using logger along with `logger_context `and specifying the `extra` argument in case we have more overwrites of the `logger_context`'s dictionary items in the future? Or put a flag near where logger_context is first created?